### DISCUSSION TITLE
tinyusb can be support the blackpill stm32f411ceu6

### DISCUSSION THREAD

**Discussion by skylin008** [adafruit/Adafruit_TinyUSB_Arduino#259] [Q&A]
**Is your feature request related to a problem? Please describe.**
Blackpill base on stm32f411ceu6 support the tinyusb

**Describe the solution you'd like**
Can be support usb msc cdc hid functions.

**Describe alternatives you've considered**
None

**Additional context**
None

**Comment by x893** [comment]
Try change EP numeration in usb_descriptors.c (cdc_uac2 example)
```
#else
	#define EPNUM_AUDIO_IN    0x83
	#define EPNUM_AUDIO_OUT   0x03

	#define EPNUM_CDC_NOTIF   0x82
	#define EPNUM_CDC_OUT     0x01
	#define EPNUM_CDC_IN      0x81
#endif

```
